packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -51,6 +60,11 @@ contract Attestations is IAttestations, Ownable, Initializable, UsingRegistry, R
     uint256 value
   );
 
+  event AccountAuthorizedAttestorSet(
```

Comment:
consider `AttestorAuthorized` for consistency with https://github.com/celo-org/celo-monorepo/pull/1177/files